### PR TITLE
feat: set vuetify on context

### LINKED ISSUE #47: $vuetify is not present in context
Thanks for your great work. `$vuetify` is not present in nuxt context. if we try to see in mounted hook, it's working
```
mounted() {
    console.log(this.$vuetify);
}
```
if we want to see in plugin, $vuetify is undefined
```
export default function({$vuetify}) {
  console.log($vuetify);
}
```
or
```
export default function({app}) {
  console.log(app.$vuetify);
}
```
if we check all context, $vuetify is not there
```
export default function(cxt) {
  console.log(cxt);
}
```

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
Resolve #47

### CODE REVIEW

File: lib/plugin.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ import Vuetify from 'vuetify'
 
 Vue.use(Vuetify)
 
-export default ({ app }) => {
-  app.vuetify = new Vuetify(<%= JSON.stringify(options.vuetifyOptions, null, 2) %>)
+export default (ctx) => {
+  const vuetify = new Vuetify(<%= JSON.stringify(options.vuetifyOptions, null, 2) %>)
+
+  ctx.$vuetify = vuetify
+  ctx.app.vuetify = vuetify;
```
Comment (pi0) [inline lib/plugin.js]: It could be better to use inject instead of directly attaching to app.

Comment (ricardogobbosouza) [reply lib/plugin.js]: With `inject`:
```
TypeError
Cannot set property $vuetify of #<Vue> which has only a getter
```

Comment (kevinmarrec) [reply lib/plugin.js]: Also got this error using inject.
I already have done this PR locally, but then figured out that I needed to provide typings for typescript, but that `$vuetify` on Vue instances has `any` type around Vuetify typings, I'll help them to make it better defined

Comment (pi0) [reply lib/plugin.js]: Hmm OK. Seems vuetify plugin also self registers!

Comment (kevinmarrec) [reply lib/plugin.js]: @pi0 Yes, but their register is the Vuetify installed instance, meanwhile `app.vuetify` & `ctx.$vuetify` are options. But anyway I tried locally and succeed to register a method helper that switch the dark mode in a plugin when `ctx.$vuetify` is set. So I think everything is good as it's now. Will work later on types.

**Comment by kevinmarrec** [comment]
Thanks @ricardogobbosouza , I'll add types later.

**Pull request merged**